sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/build.ps1

```diff
@@ -0,0 +1,11 @@
+$packageSuffix = "dev" + [datetime]::UtcNow.Ticks.ToString()
```

Comment:
I am assuming this file is only for testing purposes and will be deleted before merging?

Reply:
yup